### ISSUE TITLE
Cleanup: fix `PropTypes` for `children` declarations.

### ISSUE THREAD

**Issue by dvoytenko** [GoogleForCreators/web-stories-wp#1376]
We use many different declarations for `propTypes = {children: ...}` such as:

```
children: PropTypes.oneOfType([
   PropTypes.arrayOf(PropTypes.node),
  PropTypes.node,
]).isRequired

// or

children: StoryPropTypes.children.isRequired,
```

It appears that we may be able to simply do:

```
children: PropTypes.node
```

And validator would understand Node/Array semantics in this case.

**Comment by swissspidy** [comment]
@dvoytenko Were you thinking of a once-off fix here, or a way to enforce this?

**Comment by swissspidy** [comment]
Did a quick fix in #1377

**Comment by dvoytenko** [comment]
IMHO one-time is enough. We really all copy-paste it from each other.
